_data/advisory.yml

```diff
@@ -0,0 +1,76 @@
+- group: advisory
+  members:
+
+# - title: Quantitative MRI (qMRI)
+#   leads:
+    - name: Gilles de Hollander
+      git: Gilles86
+    - name: Kirstie Whitaker
+      git: KirstieJane
+
+# - title: Common Derivatives
+#   leads:
+    - name: Chris Markiewicz
+      git: effigies
+
+# - title: Arterial Spin Labeling (ASL)
+#   leads:
+    - name: Henk-Jan Mutsaerts
+      git: HenkMutsaerts
+    - name: Patricia Clement
+      git: patsycle
+    - name: Jan Petr
+      git: jan-petr
+    - name: Marco Castellaro
+      git: marcocastellaro
+
+# - title: Electroencephalography (EEG)
+#   leads:
+    - name: Cyril Pernet
+      git:  CPernet
+    - name: Stefan Appelhoff
+      git:  sappelhoff
+    - name: Robert Oostenveld
+      git:  robertoostenveld
+
+# - title: Hierarchical Event Descriptor (HED) Tags
+#   leads:
+    - name: Kay Robbins
+      git: vislab
```

Comment:
and @dungscout96 as co-lead

https://github.com/bids-standard/bids-website/issues/344#issuecomment-1796382329